VERSION-2-MIGRATION.md

```diff
@@ -0,0 +1,15 @@
+# Vibe 2 Migration
+
+## What's changed?
+
+- CSS modules migration - we are now using css modules instead of global css classes, this allows us to have better encapsulation and fewer conflicts with other css classes.
+- `data-testid` attributes were added, so you can target components by them instead of global class names.
+
+## Why migrate?
+
+Version 1 won't receive new features, only critical bug fixes - so if you want to use new features you should migrate to version 2.
+
+## How to migrate?
```

Comment:
The section should also include code examples. For example, for the data-testid:

Before:
```diff
- document.querySelector('monday-style-button').click();
```

After:
```diff
<Button data-testid="my-button"/>
+ document.querySelector(['data-testid="my-button"']).click();
```

For the style overrides - split it into two. the first one will be using the global monday-style classes and the second is overriding CSS directly which will no longer work.

Also, add here the topics from my comment above.

Reply:
done